build-samples/local.conf.sample

```diff
@@ -231,6 +231,6 @@ DISTRO_FEATURES_BACKFILL_CONSIDERED += "gobject-introspection-data"
 # to consider it provided.
 ASSUME_PROVIDED += "gobject-introspection gobject-introspection-native"
 
-IMAGE_INSTALL_append += " ca-certificates iptables libnss-mdns nodejs rsync"
+IMAGE_INSTALL_append += " ca-certificates iptables libnss-mdns nodejs rsync openssl"
```

Comment:
What depends on this?

Reply:
@agherzan 

```
May 28 08:48:27 e9c7d32 resin-supervisor[1054]: /usr/sbin/update-ca-certificates: line 221: openssl: not found
```

Reply:
In that case, it should be a dependency on that package. And checking the `warrior` repository I see that it is already there: http://git.yoctoproject.org/cgit.cgi/poky/tree/meta/recipes-support/ca-certificates/ca-certificates_20190110.bb?h=warrior#n85 . But... this is broken because openssl was repackaged in openssl-bin (see https://github.com/balena-os/meta-balena/pull/1509 for more details). In summary, the way I would fix this is with a bbappend on ca-certificates that does a `RDEPENDS += "openssl-bin"`.